docs/getting-into-the-code/guidelines-for-setting-up-a-local-workspace/intellij-13-code-style.md

```diff
@@ -12,7 +12,8 @@ Install the [CheckStyle-IDEA plugin](http://plugins.jetbrains.com/plugin/1065?pr
 Navigate to **File > Settings... > Plugins"**.
 On the top, click on "Marketplace".
 Then, search for "Checkstyle".
-Click on "Install" choose "CheckStyle-IDEA".
+Click on "Install" choose "CheckStyle-IDEA".<br>
```

Comment:
`<br>` is like <kbd>Shift</kbd>+<kbd>Enter </kbd>in word.

Did you experiment with 

- a new paragraph
- Using `>` (and a new paragraph)
- Using the "note" mark? (This is too heavyweight - see https://just-the-docs.com/#getting-started). However, it is exactly meant for notes.

Reply:
Today I learnt: Two spaces in markdown adds a new line.
https://www.markdownguide.org/basic-syntax/#line-break-best-practices

Reply:
> Today I learnt: Two spaces in markdown adds a new line. [markdownguide.org/basic-syntax/#line-break-best-practices](https://www.markdownguide.org/basic-syntax/#line-break-best-practices)

But very hidden in the editor

Also with automatic removel of white spaces at the end. Therefore, we have the special editor config: https://github.com/JabRef/jabref/blob/95a81e8bd7837a1b0b7fa4ae252ea3b816ad8cb9/.editorconfig#L12

Reply:
Ah, still: two spaces are shift+enter in Word. Which is a no-go.